signwriting_animation/diffusion/tests/test_dataset.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from signwriting_animation.data.data_loader import DynamicPosePredictionDataset
+
+
+@pytest.fixture(scope="module")
+def pose_dataset():
+    num_past_frames_max = 8
+    num_future_frames_max = 5
+    num_keypoints = 586
+    num_dims_per_keypoint = 3
+
+    dataset = DynamicPosePredictionDataset(
+        data_dir='signwriting_animation/data/test_data/pose_samples',
+        csv_path='signwriting_animation/data/test_data/transcription_samples/data_subset.csv',
+        num_past_frames=num_past_frames_max,
+        num_future_frames=num_future_frames_max,
+        with_metadata=False,
+        split='train'
+    )
+    return dataset, num_past_frames_max, num_future_frames_max, num_keypoints, num_dims_per_keypoint
+
+
+def test_get_batch_from_dataset(pose_dataset):
+    dataset, num_past_frames_max, num_future_frames_max, num_keypoints, num_dims_per_keypoint = pose_dataset
+
+    batch = dataset[0]
```

Comment:
this is not a batch, since it is coming from the dataset and not a data loader. it is a `datum` or something

Reply:
`sample`?